app/src/common/chromium/com/igalia/wolvic/browser/api/impl/SessionImpl.java

```diff
@@ -43,6 +45,9 @@ public class SessionImpl implements WSession {
     PanZoomControllerImpl mPanZoomController;
     private ActivityWindowAndroid mWindowAndroid;
     private IntentRequestTracker mIntentRequestTracker;
+    private Tab mTab = null;
```

Comment:
No need for null, that's the default for Java objects

Reply:
Done